### ISSUE TITLE
Traceback error when running a local scan with --json-time

### ISSUE THREAD

**Issue by pabloest** [semgrep/semgrep#2855]
**Describe the bug**
Semgrep exits before completing the scan and displays a traceback error.

**To Reproduce**
Run `semgrep --json --json-time --config=p/security-audit` on the repo at [https://github.com/rapid7/hackazon](https://github.com/rapid7/hackazon)

**Expected behavior**
Semgrep should not exit before completing its scan and should not display a traceback error.

**Screenshots**
```
🍟  semgrep --json --json-time --config=p/security-audit                                                                              Tue Mar 30, 2021 @ 09⁞05
using config from https://semgrep.dev/p/security-audit. Visit https://semgrep.dev/registry to see all public rules.
downloading config...
running 246 rules...
 54%|██████████████████████████████████████████████████████████████████████████████▉                                                                   |133/246
Traceback (most recent call last):
  File "/usr/local/bin/semgrep", line 33, in <module>
    sys.exit(load_entry_point('semgrep==0.44.0', 'console_scripts', 'semgrep')())
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/__main__.py", line 17, in main
    cli()
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/cli.py", line 457, in cli
    semgrep.semgrep_main.main(
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/semgrep_main.py", line 258, in main
    ) = CoreRunner(
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/core_runner.py", line 562, in invoke_semgrep
    ) = self._run_rules(rules, target_manager, profiler)
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/core_runner.py", line 508, in _run_rules
    rule_matches, debugging_steps, errors, rule_targets = self._run_rule(
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/core_runner.py", line 390, in _run_rule
    output_json = profiler.track(
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/profile_manager.py", line 17, in track
    result = callable(*args, **kwargs)
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/spacegrep.py", line 105, in run_spacegrep
    ) + _extract_matching_time(output_json)
  File "/usr/local/Cellar/semgrep/0.44.0/libexec/lib/python3.9/site-packages/semgrep/spacegrep.py", line 27, in _extract_matching_time
    res = json["time"]["targets"][0]["match_time"]
IndexError: list index out of range
```

**What is the priority of the bug to you?**
It is not blocking me.

**Environment**
Local Semgrep build, version 0.44.0.

**Comment by mjambon** [comment]
Thanks @pabloest. It looks like `json["time"]["targets"][0]` might be the culprit.

**Comment by nbrahms** [comment]
@DrewDennison to what level are we supporting this flag?

**Comment by ievans** [comment]
Confirmed fixed in 0.61, possibly earlier.
